sql/errors.go

```diff
@@ -130,6 +130,9 @@ var (
 	// ErrTruncateReferencedFromForeignKey is returned when a table is referenced in a foreign key and TRUNCATE is called on it.
 	ErrTruncateReferencedFromForeignKey = errors.NewKind("cannot truncate table %s as it is referenced in foreign key %s on table %s")
 
+	// ErrInvalidPrecisionSyntax is returned when a column type-definition has argument violations.
+	ErrInvalidPrecisionSyntax = errors.NewKind("column %s type definition is invalid: %s")
```

Comment:
The error variable name is more specific than the error message. I'd rename it to `ErrInvalidColTypeDefinition` based on the error message.

Reply:
👍 Yeah makes sense